code-challenges.test.js

```diff
@@ -33,9 +118,35 @@ const nums1 = [3, 56, 90, -8, 0, 23, 6]
 const nums2 = [109, 5, 9, 67, 8, 24]
 // Expected output: [5, 109]
 
+// describe("minAndMax", () => {
+//   const expected = [-8, 90,]
+//   const expected2 = [5, 109]
+//   it("returns an array of the minimum and maximum numbers in that order.", () => {
+//     expect(minAndMax(nums1)).toEqual(minAndMax(expected))
+//     expect(minAndMax(nums2)).toEqual(minAndMax(expected2))
+//   })
+// })
+// ReferenceError: minAndMax is not defined
+// Pass
 
 // b) Create the function that makes the test pass.
 
+// Pseudo code:
+// Declare a function call minAndMax
+// Parameter: numArray
+// Delcare a variable called newArray. Give newArray the value [Math.min(...numArray), Math.max(...numArray)]
+// Return newArray
+
+
+const minAndMax = (numArray) => {
+    let newArray = [Math.min(...numArray), Math.max(...numArray)]
+    return newArray
+}
+
+// console.log(minAndMax(nums1)) // [ -8, 90 ]
+// console.log(minAndMax(nums2)) // [ 5, 109 ]
+
```

Comment:
Again, excellent work! Your test is on point.  Your code is elegant.  Your pseudocode could be more explanatory around what Math.min() and Math.max() are doing.  Overall, great job!